testsuite/do_tests.sh.in

```diff
@@ -637,8 +642,9 @@ fi
 #        echo          "Running test '${test_name}' with $np $proc_txt... " >> "${TEST_LOGFILE}"
 #        printf '%s' "  Running test '${test_name}' with $np $proc_txt... "
 #
-#        # Copy everything to the tmpdir.
-#        cp "${music_file}" "${sh_file}" "${sli_files}" "${tmpdir}"
+#        # Copy everything to the tmpdir. Not all files have to exist and therefore variables can also be empty.
+#        # To prevent 'cp' from terminating, exit code and error message is suppressed.
```

Comment:
```suggestion
#        # Copy everything to the tmpdir. As some files might not
#        # exist, variables can also be empty. To prevent 'cp' from
#        # terminating on non-existent files, its exit code and error
#        # message are suppressed.
```